maths/validate/validate.go

```diff
@@ -96,12 +96,16 @@ func scaleMultiPolygon(p tegola.MultiPolygon, factor float64) (bmp basic.MultiPo
 	return bmp
 }
 
-func CleanGeometry(ctx context.Context, g tegola.Geometry, extent *geom.Extent) (geo tegola.Geometry, err error) {
+func CleanGeometry(ctx context.Context, g tegola.Geometry, extent *geom.Extent, clipGeometry bool) (geo tegola.Geometry, err error) {
```

Comment:
remove `clipGeometry` parameter and instead add a comment describing the use of a nil extent to CleanGemetry.